config/schema/contact_tab.schema.yml

```diff
@@ -22,6 +22,9 @@ contacts.contact_tab.*:
       sequence:
         type: contacts.relationship.[%name]
         label: 'Relationship'
-    block:
-      type: contacts.block_plugin.[id]
-      label: 'Block'
+    blocks:
+      type: sequence
+      label: 'Relationships'
+      sequence:
+        type: contacts.block_plugin.[id]
+        label: 'Blocks'
```

Comment:
This doesn't look right - Relationships is definitely wrong, but we've also got a label at both levels. Is that right and, if so, should the lower one be singular?